Makefile

```
@@ -154,12 +159,17 @@ image-build-%: check-container-tool ## Build Docker image ## Build Docker image
  		-t $($*_IMAGE) -f Dockerfile.$* .
 
 .PHONY: image-push
-image-push: image-push-epp image-push-sidecar ## Push Docker images to registry
+image-push: epp-image-push sidecar-image-push ## Push container images to registry
 
-.PHONY: image-push-%
-image-push-%: check-container-tool ## Push Docker image to registry
-	@printf "\033[33;1m==== Pushing Docker image $($*_IMAGE) ====\033[0m\n"
-	$(CONTAINER_TOOL) push $($*_IMAGE)
+.PHONY: %-image-push
+%-image-push: check-container-tool load-version-json ## Push container image to registry
```

You accidentally undone important fixes on these lines.
```suggestion
.PHONY: mage-push-%
image-push-%: check-container-tool ## Push container image to registry
```